list-credential.groovy

```diff
@@ -0,0 +1,14 @@
+/**
+Author: kuisathaverat
+Description: List ID and Description of all credentials on a Jenkins Instance.
```

Comment:
Rather: "List ID and Description of all root credentials" no ? I think that if you lookup with the context of `Jenkins.instance`, you'll only see root credentials.